test/test-StatsLogger.js

```diff
@@ -410,5 +410,43 @@ describe('StatsLogger', function() {
     }, 1050);
   });
 
+  it('should be able to add a captureFunction', function(done) {
+    var a = statsLogger.createInstance(60, backend, {}, {});
+    a.addStat("myStat", "counter", {initialValue: 0});
```

Comment:
You should use single quotes for consistency (I believe our lint config even usually includes a setting to check for this).